content/zh/seachat/seachat-manual/04-channels/04-seachat-whatsapp-integration.md

```diff
@@ -0,0 +1,515 @@
+</a>
+    <p style="margin-top: 20px; font-size: 15px">Account information</p>
+</div>
+</div>
+
+<br/> 
+
+這將帶您進入 **WhatsApp Business Manager** 下的正確帳戶。在這裡，您可以進行必要的更改，上傳個人資料圖片，並保存更新。這些更改可能需要幾分鐘才能生效。一旦生效，當您訪問 WhatsApp 商業帳戶的信息頁面時，您會注意到更新的個人資料信息。
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+<a href="/images/seachat/zh/channels/whatsapp/deploy-2.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/zh/channels/whatsapp/deploy-2.png" alt="">
+</a>
+    <p style="margin-top: 20px; font-size: 15px">Make Changes to Account</p>
+</div>
+</div>
+
+<br/> 
+
+幾分鐘後，如果您在手機上的 WhatsApp 應用中點擊您的 WhatsApp 商業帳戶的信息頁面，它將反映出這些更改：
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+<a href="/images/seachat/zh/channels/whatsapp/deploy-3.png" target="_blank">
+<img width="50%" height="auto" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/zh/channels/whatsapp/deploy-3.png" alt="">
+</a>
+    <p style="margin-top: 20px; font-size: 15px">WhatsApp Business Account Info</p>
+</div>
+</div>
+
+<br/> 
+
+
+優化您的 WhatsApp 商業個人資料不僅僅是添加個人資料圖片。以下是一些增強個人資料的額外提示：
+
+2. **使用清晰專業的個人資料圖片**：
+選擇能代表您的品牌或業務的高質量圖片。避免模糊或低解析度的圖片，因為它們會給人負面印象。
+
+3. **添加相關聯絡信息**：
+確保您的電話號碼、電子郵件地址和網站（如適用）在個人資料中顯眼顯示。這讓客戶可以通過他們喜歡的渠道輕鬆聯繫您。
+
+4. **使用標籤和標記**：
+使用標籤和標記來組織您的聯絡人和對話。這有助於您有效地分類和管理客戶互動，使您更容易跟踪和回應特定查詢或請求。
+
+通過遵循這些提示，您可以正確地優化您的 WhatsApp 商業帳戶，創建專業的個人資料，並增強整體的客戶參與度。
+
+# 與真人助理互動
+您是否注意到在上面的圖片中我使用 /live_agent 來請求真人助理？如果助理通過設置他們的在線狀態在線：
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 60%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+  <a href="/images/seachat/zh/channels/whatsapp/live-agent-status.png" style="height: 200px; width: 100%; height: 100%;display: flex; justify-content: center; align-items: center; overflow: hidden;" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/zh/channels/whatsapp/live-agent-status.png" alt="">
+</a>
+    <p style="margin-top: 20px; font-size: 15px">Live agent status
+</p>
+</div>
+</div>
+
+<br/> 
+
+他們可以直接與用戶對話！
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+  <a href="/images/seachat/zh/channels/whatsapp/live-agent-interaction.png" style="height: 200px; width: 100%; height: 100%;display: flex; justify-content: center; align-items: center; overflow: hidden;" target="_blank">
```

Comment:
this is a picture inherited from the En page. but it's a wrong picture showing just FB Messenger bot. can you make a WhatsApp picture? You can use the whatsapp bot on seaslat.ai homepage to trigger the /live_agent command